packages/amplify_analytics_pinpoint_plugin/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -9,6 +9,8 @@ environment:
 
 dependencies:
   amplify_analytics_plugin_interface: 0.4.0
+  amplify_analytics_pinpoint_android: 0.0.1
```

Comment:
We've decided to update these versions to match everything else in our library, for consistency.